### DISCUSSION TITLE
is the snapping for chatterino messed up or is it something on my end?

### DISCUSSION THREAD

**Discussion by Tyr808** [Chatterino/chatterino2#3689] [Q&A] [answered]
I've noticed that chatterino doesn't have the default windows 11 snapping tools when mousing over the maximize icon on the top bar like most windows do in 11. If you force snapping via FancyZones, the chatterino window is slightly too big on 11 (as if it's not accounting for rounded edges?)

Sorry in advance if this is the wrong place, I'm new to github for anything other than simply downloading stuff and I've tried googling this but it seems to be too vague and too small of a problem.

Anyway, I'd be curious if other people are having the same issue or not. Absolutely not a big deal at all, just a minor inconvenience when using tools to organize your screens. As tiny as this issue is it would be so satisfying to not have to keep manually adjusting this or put up with an uneven window.

Thanks in advance for such an amazing piece of software, cheers!

**Comment by Felanbird** [comment]
https://github.com/Chatterino/chatterino2/issues/2205#issuecomment-868001183

**Reply by Felanbird** [reply]
But as a comment - we haven't really taken any steps to look into Windows 11 support, so if there's something other than the comment I've noticed (rounded edges you suggested), it might just have to wait.

**Reply by Tyr808** [reply]
> [#2205 (comment)](https://github.com/Chatterino/chatterino2/issues/2205#issuecomment-868001183)

Thank you for this! this does actually reduce the border. oddly enough it's still about 1 or 2 pixels larger than it should be, but it's small enough that I can ignore this and it's not spilling over to the other monitor at least.

As for fixes, I wanted to make sure this was known as it's been a mild annoyance, but I also don't want to sound like I'm demanding anything here as I'm well aware that I'm using free software here. Thanks gain for the speedy reply and bringing the workaround to my attention :)

**Comment by pajlada** [comment, marked as answer]
I believe this has been fixed. If that's not the case, let me know!
